app/helpers/kaui/date_helper.rb

```diff
@@ -13,7 +13,7 @@ def format_date(date, timezone="Pacific Time (US & Canada)")
 
       # If not, convert into account timezone and return the date part only
       parsed_date = DateTime.parse(date.to_s).in_time_zone(timezone)
-      parsed_date.to_s(:date_only)
+      parsed_date.to_s(:pretty)
```

Comment:
> Timezone could be annoying thing. I'll create email about requirements of time for customer support this morning (PDT).

If we really need to support a timezone override, we could introduce a new config option to override all `DateTime` timezone.

> [transaction_date = Date.parse(transaction.effective_date)](https://github.com/killbill/killbill-admin-ui/blob/8bb930ecbdbe5bcefe568cb3f7c37ba914e8504f/app/controllers/kaui/payments_controller.rb#L41
) 
[format_date(transaction.effective_date, account.time_zone).html_safe ](https://github.com/killbill/killbill-admin-ui/blob/c59c681fd6f1ecb9e1eaef60b040411946a52922/app/views/kaui/payments/_payment_table.html.erb#L84)

In both cases, `format_date` shouldn't be in use anymore. Instead, we should introduce a new `format_datetime` (similar signature). A default timezone would be used (e.g. PDT) if configured, account timezone otherwise.